katago_server/distributed_efforts/tests/test_api.py

```diff
@@ -26,28 +42,68 @@ def test_get_job_anonymous(self):
         """
         An anonymous user should connect to get a new task
         """
-        # Given
```

Comment:
Oh, I liked the "Given when Then" comments. It helped me structure the test. 
cf https://martinfowler.com/bliki/GivenWhenThen.html

Not strongly for keeping them but any reason to delete the existing ?

Reply:
Hehe, interesting. They seemed unnecessary to me, sort of like
```
# Initialize x
x = 0
# Add one to x
x += 1
# Assert that x is positive
assert x > 0
```
Because as far as I can tell, "then" simply meant "here is where we begin to assert things", but you can see in the code itself that it's asserting things. And for there seemed to be not a clear distinction between "given" and "when" in our situation - for example why is `client.login(username="test", password="test")` part of "given" but not part of "when"? If "when" is always simply the last command before we begin asserting things or whose return value we are asserting things about, then it is also redundant, I didn't see why there needs to be a comment saying "this is the last command before we begin to assert things", you can just see that it is the last command before asserting things and/or that the assertions are all about the response to that command.